nixos/modules/services/networking/nomad.nix

```diff
@@ -52,12 +52,25 @@ in
       settings = mkOption {
         type = format.type;
         default = {
-          # Agrees with `StateDirectory = "nomad"` set below.
           data_dir = "/var/lib/nomad";
         };
         description = ''
           Configuration for Nomad. See the <link xlink:href="https://www.nomadproject.io/docs/configuration">documentation</link>
           for supported values.
+
+          Notes about `data_dir`:
+
+          If `data_dir` is set to a value other than the default value of
+          `"/var/lib/nomad"` it is the Nomad cluster manager's responsibility
+          to make sure that this directory exists and has the appropriate
+          permissions. One way to ensure this is the case to create the
+          directory and adjust its permissions as needed using the
+          `ExecStartPre` feature of systemd.
```

Comment:
I think this sentence sounds a bit weird, "One way to ensure this is the case..."?

Reply:
I'll remove it.